src/ResourceManager/DataMigration/AzureRM.DataMigration.Netcore.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.11.0'
```

Comment:
@hitenjava please revert the change made to this version -- we handle versioning of modules at the time of the release

Reply:
Done